lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.json

```diff
@@ -16,9 +16,15 @@
           },
           "publish_ports": {
             "label": "Container ports to publish",
-            "override": false,
+            "override": true,
             "type": "text",
             "tip": "Comma-separated list of container ports to publish"
+          },
+          "command": {
+            "label": "Command to run, including arguments",
```

Comment:
I used the same nomenclature as Docker, itself. This is the command that is sent to the entrypoint script, if present, in the container. The pretty standard "docker-compose" tool uses "command" in its configs.

https://docs.docker.com/engine/reference/builder/#cmd

While it is technically a set of additional arguments to an entrypoint script (or to sh, if no entrypoint), most docker users expect it to be a command + arguments to that command.